src/thruster_types/thruster_types.proto

```diff
@@ -1,6 +1,19 @@
 package rock_thruster.msgs;
 
-message ThrusterInput
+message Raw
 {
-    optional double     frequency = 1 [default = 0];
+    optional string         name = 1;
```

Comment:
In general, would it not be enough (and better ...) to have

~~~
Thruster
{
   optional string name
   optional double raw;
   optional double effort;
}
~~~